### PR TITLE
test invalid output values

### PR DESCRIPTION
**Pull request by emord**
http://manage.dimagi.com/default.asp?236968#1215641

buddy @proteusvacuum

### CODE REVIEW

**Comment by proteusvacuum** [comment]
Will this notify the user that there is an error?
(sorry, not that familiar with vellum code)

**Comment by emord** [comment]
this is some bad ux, but we have never had any warnings about invalid xpaths except for in the advanced xpath editor. manual output values are a fairly advanced thing now that easy references exist anyways.

**Comment by proteusvacuum** [comment]
👍

File: src/javaRosa/util.js [hunk 1]
```diff
@@ -270,8 +270,14 @@ define([
...
                     var $this = $(this),
-                        value = xpathParser.parse($this.attr('value') || $this.attr('ref'));
-                    $this.attr('value', value[functionName]());
+                        value = $this.attr('value') || $this.attr('ref'),
+                        parsedValue;
+                    try {
+                        parsedValue = xpathParser.parse(value);
+                        $this.attr('value', parsedValue[functionName]());
+                    } catch (e) {
+                        $this.attr('value', value);
```
Comment (millerdev) [inline src/javaRosa/util.js]: Should this use the `#invalid/xpath ` prefix?

Comment (emord) [inline src/javaRosa/util.js]: I think that should only be added to attributes with a `vellum:` namespace correct? We could add a `vellum:value` to this output value, but I don't think that's necessary.

Comment (millerdev) [inline src/javaRosa/util.js]: I guess that means you can't use hashtags (like `#case/foo`) in typed output value expressions?

**Pull request merged**